coalib/coala_modes.py

```diff
@@ -66,6 +70,20 @@ def mode_json(args):
         results, exitcode, _ = run_coala()
 
     retval = {'bears': results} if args.show_bears else {'results': results}
+
+    if args.log_json:
+        logs = []
+        for line in log_stream.getvalue().splitlines():
+            logs.append(json.loads(line))
+        retval = {'logs': logs}
```

Comment:
you override `retval` again, this does not look right^^ If you want to update `retval`, just do `retval['logs'] = logs`